packages/safe-tools-client/app/components/schedule-payment-form-action-card/ui/index.gts

```diff
@@ -305,21 +305,24 @@ export default class SchedulePaymentFormActionCardUI extends Component<Signature
               <BoxelField @label="Recipient Will Receive" data-test-summary-recipient-receives @vertical={{true}} style={{cssVar boxel-field-label-justify-content="end"}}>
                 <div class="schedule-payment-form-action-card--fees-value">
                   {{#if @paymentAmountTokenQuantity}}
-                    {{@paymentAmountTokenQuantity.displayable}}
+                    <div>{{@paymentAmountTokenQuantity.displayable}}</div>
+                    <div class="schedule-payment-form-action-card--fee-usd-estimate">{{tokenToUsd tokenQuantity=@paymentAmountTokenQuantity}}</div>
```

Comment:
In BEM `--` is used for modifier classes. Did you mean to use `__` (block element)?